### DISCUSSION TITLE
塞姆·克莱门森故居 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#481] [General]
# 塞姆·克莱门森故居 - Miao Yu | 于淼

https://yufree.cn/cn/2024/11/06/mark-twain/

**Comment by earfanfan** [comment]
哇，初中就有图书馆可以借书，好羡慕好羡慕好羡慕啊，我直到上大学才知道图书馆的存在。而且因为教材都是住在城市里的人编写的，课本里描述的很多事物是从小生活在村子里的我从没见过的，自然连想象都想象不出来了。

不过很奇怪的一点是，我上初中的时候也流行读名著，我们同学之间的方法是每个人攒钱买本书以后，大家交换着读，坑的是那时候我们那流行读的名著多是外国文学名著，那时候读《呼啸山庄》《巴黎圣母院》《钢铁是怎样炼成的》等，只能说读了，跟没读一样。

另外，你们那里游览博物馆的方式也好有趣啊，感觉国内名人故居是不会这么干的，可能商业景区才会有角色扮演。

**Reply by yufree** [reply]
初高中的图书馆其实利用率很低，但在阅览室看各种报纸杂志是我那时候最放松的消遣方式，类似现在上网看世界。
我现在可以读进去一些名著了，只能说中学生那点人生阅历读不明白是正常的，读明白的大概率也是武侠侦探言情这类，更不用说文化差异下读西方的名著还得了解历史背景，当然读进去也就那样，当代很多人的经历复杂度与作品深度已经远超名著年代了，特别是不少科幻作品里对人性的思考，没必要是古非今。

**Reply by earfanfan** [reply]
>没必要是古非今。

看到这句我第一反应是“咦，我没有是古非今呀”，太久没到你这里溜达都有点忘记你长撒样了（PS俺没见过你的躯壳，是指你的文字留给我的印象），哈，看到这句我就想起来好些，想起来你对待本读者的态度是平等和尊重的，又想起来你有喜欢讲道理的特点，然后就立刻明白你不是说我，而是你时刻随时随地的思考，写着写着就写到这里了。我还想起来，三水你还很喜欢写“没必要xxx”句式，大约就是你已经思考过很多很多事物，脑子里已经存储了很多道理，当一开始写到中学生读不明白时，自动触发联动的一整段思考，自然而然就会写到没必要是非古今了。

转念一想，只有以前上学的时候被灌输过“名著”的概念，现在看书根本都意识不到撒是名著，对了，你是咋定义名著的？如果说是名气大且作者得过世界级大奖的话，我今年好像读了好几本名著，比如略萨的《给青年小说家的信》，黑塞的《悉达多》（姜乙译本），忘记谁写的剧本《一个无政府主义者的意外死亡》，还有就是博尔赫斯的《杜撰集》。刚开始看这几本书的时候我只是接收了忘记哪里看来的无形安利，结果看着看着（PS微信读书上面能看到别人划线留言）发现这几个作者好像都是撒诺贝尔奖得主，知道作者很厉害以后，看不懂就会理所当然觉得看不懂也很正常，反正俺只是个普通人。

**Reply by yufree** [reply]
没那么复杂，是古非今说的是我对“名著”的看法。名著里的内容理应是可以接受时间检验的，但个人体验未必，名著出现的年代多是工业革命初期社会出现剧变的时代之后，很多议题核心就是对外与现代/农业社会的矛盾与调和，会对内与自己过不去或和解，这两大议题现在也不过时才被认可为名著，但人口总量制约下人类物种多样性无法完全体现，真正能出名著的时代反而可能是未来几十年直到全球人口达峰后收缩回工业革命时代水平，社会矛盾可能会出乎意料。诺奖啥的只是锦上添花，好的作品不在意是否得奖，得奖了才出名更可能是低于奖项平均水平或小圈子自嗨，普通人生活不需要专家协助审美，你觉得好那就是好。
